index.js

```diff
@@ -3,17 +3,17 @@ module.exports = function(handler) {
   if (!(handler instanceof Function || handler instanceof RegExp))
     throw new Error('Unsupported param handler. Either function or Regexp expected')
 
-  return function(req, res, next, val, name) {
-    val = req.params[name]
+  return function(req, _res, next, _val, name) {
+    const handlerValue = req.params[name];
```

Comment:
the code was introduce from this code change
https://github.com/Adslot/node-express-params-handler/pull/5/files#r45143030